Makefile

```diff
@@ -89,8 +89,8 @@ ifeq ($(detected_OS),Darwin)
  export MACOSX_DEPLOYMENT_TARGET
  PKG_TARGET := pkg-macos
  RUN_TARGET := run-macos
- QMAKE_PATH := $(shell which qmake);
- QT_ARCH := $(shell lipo -archs $(QMAKE_PATH))
+ QMAKE_PATH := $(shell which qmake)
```

Comment:
Is this even used? I think you can drop this one

Reply:
Removed! Thanks 🍻